frontend/packages/knative-plugin/integration-tests/features/knative/create-knative-workload.feature

```diff
@@ -107,3 +107,17 @@ Feature: Create a workload of 'knative Service' type resource
               And user clicks on the Knative Service workload "dancer-ex-git"
               And user clicks on name KSVC "dancer-ex-git"
              Then user will see value of autoscaling.knative.dev/maxScale, autoscaling.knative.dev/minScale, autoscaling.knative.dev/target, autoscaling.knative.dev/targetUtilizationPercentage, autoscaling.knative.dev/window under annotation and containerConcurrency under spec as under spec 5, 1, 3, 70, 9s and 15 respectively
+
+
+        @regression @manual
+        Scenario Outline: knative resource type in upload JAR file
```

Comment:
It should be Scenario, when there is no examples

Reply:
Updated